iidm/iidm-xml-converter/src/test/java/com/powsybl/iidm/xml/TerminalRefTest.java

```diff
@@ -24,7 +27,10 @@
     public void roundTripTest() throws IOException {
         String filename = "terminalRef.xiidm";
 
-        Network network = Importers.loadNetwork(filename, getClass().getResourceAsStream("/" + filename));
+        LocalComputationConfig config = new LocalComputationConfig(tmpDir, 1);
+        ComputationManager computationManager = new LocalComputationManager(config);
```

Comment:
You should use mockito for ComputationMock, it is never call but if one day it is the case it will throw an exception.